src/llm/llm_executor.hpp

```diff
@@ -61,7 +62,7 @@ struct LLMExecutor {
 #pragma GCC diagnostic ignored "-Wunused-but-set-variable"
     void printMetrics() {
         PipelineMetrics metrics = pipe->get_metrics();
-        SPDLOG_LOGGER_DEBUG(llm_executor_logger, "All requests: {}; Scheduled requests: {}; Cache usage {:.1f}%;",
+        SPDLOG_LOGGER_INFO(llm_executor_logger, "All requests: {}; Scheduled requests: {}; Cache usage {:.1f}%;",
```

Comment:
Why info here? Wouldn't it trigger per request logging on level INFO?

Reply:
Those lines would show on DEBUG anyway - I don't see a reson to put it in INFO level.

Reply:
I have changed to log the metrics every 10th step, thus changing to info as Darek requested.